testplan/testing/multitest/driver/kafka.py

```diff
@@ -0,0 +1,112 @@
+"""
+Driver for Zookeeper server
+"""
+import os
+import re
+
+from schema import Or
+from testplan.common.config import ConfigOption
+from testplan.common.utils.path import (
+    makeemptydirs,
+    makedirs,
+    instantiate,
+    StdFiles,
+)
+from testplan.common.utils import context
+from testplan.testing.multitest.driver import app
+
+KAFKA_START = "/opt/kafka/bin/kafka-server-start.sh"
+
+
+class KafkaStandaloneConfig(app.AppConfig):
+    """"
+    Configuration object for
+    :py:class:`~testplan.testing.multitest.driver.kafka.KafkaStandalone` resource.
+    """
+
+    @classmethod
+    def get_options(cls):
+        return {
+            ConfigOption("cfg_template"): str,
+            ConfigOption("zookeeper_connect"): Or(str, context.is_context),
```

Comment:
If this is only used in config file, maybe we don't need this ConfigOption? We can refer to zk driver's property directly in kafka's config file.